lunatrace/bsl/hasura/migrations/lunatrace/1656548293311_use_new_vulnerability_table/up.sql

```diff
@@ -0,0 +1,35 @@
+ALTER TABLE "public"."findings" DROP CONSTRAINT "findings_vulnerability_id_fkey",
+  ADD CONSTRAINT "findings_vulnerability_id_fkey"
+  FOREIGN KEY ("vulnerability_id")
+  REFERENCES "vulnerability"."vulnerability"
+  ("id") on update no action on delete no action;
+
+ALTER TABLE "public"."ignored_vulnerabilities" DROP CONSTRAINT "ignored_vulnerabilities_vulnerability_id_fkey",
+  ADD CONSTRAINT "ignored_vulnerabilities_vulnerability_id_fkey"
+  FOREIGN KEY ("vulnerability_id")
+  REFERENCES "vulnerability"."vulnerability"
+  ("id") ON UPDATE NO ACTION ON DELETE CASCADE;
+
+ALTER TABLE "public"."findings" ALTER column "vulnerability_id" SET NOT NULL;
+
+ALTER TABLE "public"."guide_vulnerabilities" DROP CONSTRAINT "guide_vulnerabilities_vulnerability_id_fkey",
+  ADD CONSTRAINT "guide_vulnerabilities_vulnerability_id_fkey"
+  FOREIGN KEY ("vulnerability_id")
+  REFERENCES "vulnerability"."vulnerability"
+  ("id") ON UPDATE NO ACTION ON DELETE CASCADE;
+
+ALTER TABLE "vulnerability"."equivalent" ALTER column "a" SET NOT NULL;
+
+ALTER TABLE "vulnerability"."equivalent" ALTER column "b" SET NOT NULL;
+
+ALTER TABLE "vulnerability"."vulnerability" ADD COLUMN "cvss_score" REAL NULL;
+
+CREATE OR REPLACE VIEW public.default_branch_builds AS
```

Comment:
What in this view needed changing?

Reply:
there was a subquery that was returning multiple rows, but the result from the query was being used in a place where it was expected to only be returning one row. These changes remove the subquery and use joins instead.